src/applications/vaos/api/index.js

```diff
@@ -17,7 +17,9 @@ function getStagingId(facilityId) {
 }
 
 const USE_MOCK_DATA =
-  environment.isLocalhost() && !environment.API_URL.includes('review.vetsgov');
+  !window.Cypress &&
```

Comment:
Turns off mocking when Cypress is running: https://docs.cypress.io/faq/questions/using-cypress-faq.html#Is-there-any-way-to-detect-if-my-app-is-running-under-Cypress